### ISSUE TITLE
fxtwitter share button

### ISSUE THREAD

**Issue by FinecoFinit** [revanced/revanced-patches#2422]
### Application

Twitter

### Issue

Embed fixing 

### Patch

Use stock share button or add another one for fxtwitter/vxtwitter/etc

### Motivation

Sharing with other networks should be granted to community

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by oSumAtrIX** [comment]
FxTwitter id discontinued

**Comment by JodanJodan** [comment]
@oSumAtrIX fxTwitter changed hands over a year ago; it's [actively developed](https://github.com/FixTweet/FixTweet), not discontinued.

**Comment by oSumAtrIX** [comment]
I am not sure if you understood the repository you linked. The repository is not about a Twitter client.

**Comment by oSumAtrIX** [comment]
Also, the repository is open-source, nor an Android application, which is why no patch is necessary

**Comment by JodanJodan** [comment]
Yes, I understand what the repository is, it's for the [fxtwitter.com](https://fxtwitter.com/) service the ticket creator is requesting a share button for. fxTwitter is a service that proxies and enriches Twitter embed metadata/

**Comment by oSumAtrIX** [comment]
Then I am not sure why you think ReVanced can patch it, considering it is not an Android app.

**Comment by JodanJodan** [comment]
Ticket creator is requesting a patch in the Twitter Android app for a share button using [fxtwitter.com](https://fxtwitter.com/) links instead of [twitter.com](https://twitter.com/), not a patch for fxTwitter.

**Comment by FinecoFinit** [comment]
Also, there are more instances of fxtwitter like vxtwitter etc, just make the ability to change the link

**Comment by kairusds** [comment]
Other alternative links can also be vxtwitter.com and nitter.com

**Comment by FrozenAlex** [comment]
nitter seems to be dead
